docs/source/conf.py

```diff
@@ -15,9 +15,9 @@
 
 from pallets_sphinx_themes import ProjectLink
 
-import build
+sys.path.insert(0, os.path.abspath(os.path.join(__file__, '..', '..', '..')))  # isort:skip
```

Comment:
Generally, you should not be injecting paths manually on sys.path. Consider instead making the docs environment install the package and just import. See https://github.com/pypa/virtualenv/blob/master/.readthedocs.yml#L8-L14 for the readthedocs incantation